en/getting_started/flight_controller_selection.md

```diff
@@ -1,36 +1,43 @@
 # Flight Controller Selection
 
+Flight controllers are the "brains" of an unmanned vehicle.
+PX4 can run on [many flight controller boards](../flight_controller/README.md).
+
 You should select a board that suits the physical constraints of your vehicle, the activities you wish to perform, and of course cost.
 
-PX4 can run on many flight controller boards (see [Autopilot Hardware](../flight_controller/README.md), or the list of supported boards [here on Github](https://github.com/PX4/PX4-Autopilot/#supported-hardware)).
-A subset of the available options are listed below.
+<img src="../../assets/flight_controller/pixhawk6x/pixhawk6x_hero_upright.png" width="130px" title="Holybro Pixhawk6X"> <img src="../../assets/flight_controller/cuav_pixhawk_v6x/pixhawk_v6x.jpg" width="230px" title="CUAV Pixhawk 6X" > <img src="../../assets/flight_controller/cube/orange/cube_orange_hero.jpg" width="300px" title="CubePilot Cube Orange" />
+
 
 ## Pixhawk Series
 
 [Pixhawk Series](../flight_controller/pixhawk_series.md) open-hardware flight controllers run PX4 on NuttX OS.
 With many form factors, there are versions targeted towards many use cases and market segments.
 
-The following [Pixhawk Standard Autopilots](../flight_controller/autopilot_pixhawk_standard.md) are supported and tested by the PX4 maintenance and test teams (other autopilots are [manufacturer-supported](../flight_controller/autopilot_manufacturer_supported.md)).
+[Pixhawk Standard Autopilots](../flight_controller/autopilot_pixhawk_standard.md) are used as the PX4 reference platform.
+They are supported and tested by the PX4 development team, and are highly recommended.
 
-Controller | Description
```

Comment:
Note to self, deleted all these lists. We have heaps of these and keeping them up to date is impossible. So here just provide better linking.